submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/submitter/request/SubmissionRequestLoader.kt

```diff
@@ -73,7 +73,7 @@ class SubmissionRequestLoader(
 
     private fun asCompressedFile(accNo: String, version: Int, directory: NfsFile): NfsFile {
         fun compress(file: File): File {
-            val tempFolder = fireTempDirPath.resolve("$accNo/$version")
+            val tempFolder = fireTempDirPath.resolve("$accNo/$version/${directory.filePath.substringBeforeLast("/")}")
```

Comment:
really subjective but  I think the problem is not in this line but rather `tempFolder.resolve("${file.name}.zip")` the problem with that code is that we are using fixed file path rather than do that we should use `Files.createTempFile` which will atuomatically garantee to us that same name is not used.

Reply:
It actually solves the problem but, this method uses the VM's temp folder which is only 10 GB currently which I think won't be enough for the imaging people's requirements

Reply:
what I mean is use the same method over the same folder `fireTempDirPath.createTempFile("$accNo/$version")`

Reply:
Got it, makes sense. Done!